Update documentation for triggering workspace runs with Git tags (#64)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/vcs.mdx

-> **Note:** Terraform Cloud triggers runs on all attached workspaces if it does not receive a list of changed files or if that list is too large to process. When this happens, Terraform Cloud may show several runs with completed plans that do not result in infrastructure changes.

#### Trigger runs when a git tag is published

This option instructs Terraform to begin new runs only for changes that have a specific tag format.

The tag format can be chosen between the following options:

> - **Semantic Versioning:** It matches tags in the popular [SemVer format](https://semver.org/). (e.g. 0.4.2)

Reviewer: Do we want to give the regex for these three values?

Reviewer 2: Done

> #### Trigger runs when a git tag is published
> 
> This option instructs Terraform to begin new runs only for changes that have a specific tag format.
> 
> The tag format can be chosen between the following options:
> - **Semantic Versioning:** It matches tags in the popular [SemVer format](https://semver.org/). (e.g. 0.4.2)
> - **Version contains a prefix:** It matches tags which have an additional prefix before the [SemVer format](https://semver.org/). (e.g. version-0.4.2)
> - **Version contains a suffix:** It matches tags which have an additional suffix after the [SemVer format](https://semver.org/). (e.g. 0.4.2-alpha)
> - **Custom Regular Expression:** You can define your own regex to be matched against tags.

Reviewer 3: For SemVer, do we also accept if there is a v in front of the tag? I ask because for other things (like provider versions) I believe we recognize the tag if it's SemVer and/or we accept a v in front of the SemVer tag format - e.g.,`v0.1.2`. 

```suggestion
#### Trigger runs when a git tag is published

This option instructs Terraform to begin new runs when you publish a Git tag with a specific format.

Choose one of the following options to specify the tag format:
- **Semantic Versioning:** Tags match [SemVer format](https://semver.org/). (e.g., 0.4.2).
- **Version contains a prefix:** Tags have additional prefix before the [SemVer format](https://semver.org/). (e.g. version-0.4.2)
- **Version contains a suffix:** Tags have an additional suffix after the [SemVer format](https://semver.org/). (e.g., 0.4.2-alpha)
- **Custom Regular Expression:** Define your own regex for tag format.
```

Reviewer 4: Hey @laurapacilio I would say that for that use case we offer it through the `Version contains a prefix` option.

btw. could you please explain why do we put comma behind `e.g.` in the first and third point but not in second? And what is the grammar rule driving this?

Also isn't the period at the end of the first point redundant? If it's not we should add it to the second and third as well.

Reviewer 3: Hi! Thank you for asking!

## Periods
Ah thank you! This is what I get for trying to multitask while editing - I meant to move all of the commas to the end so that it's an entire sentence with the e.g. included as an example. I've updated my suggestion to fix this.

## Commas after e.g.
It's a mistake that I didn't add one after the second example - thank you for catching that!  I've updated my suggestion to fix.

In terms of explanation: I was always taught that you add a comma after e.g., which is why I made those changes here. For example - [this article](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/#:~:text=E.g.%20and%20i.e.%20are%20both,one%20item%20in%20your%20list.) talks about how generally you add a comma. I believe it is also recommended in many style manuals, for example, I believe the Chicago Manual of Style recommends a comma after e.g. Interestingly, as I do research, I believe there is a difference here between American style guide and British English (e.g. Oxford Manual of Style) where British English often does not include the comma.

I'm going to:
- Bring this up with technical writing so we can get something official added to the HashiCorp style guide either way
- Recommend that here we use commas, to be consistent with what we have recently added to the rest of the page.

I hope this is helpful - let me know if you have more questions. I appreciate the attention to detail here and that you didn't just blindly accept my edits!
